speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/SpeechToText.java

```diff
@@ -425,10 +448,12 @@ public WebSocket recognizeUsingWebSocket(RecognizeOptions recognizeOptions, Reco
       urlBuilder.addQueryParameter("customization_id", recognizeOptions.customizationId());
     }
     if (recognizeOptions.languageCustomizationId() != null) {
-      urlBuilder.addQueryParameter("language_customization_id", recognizeOptions.languageCustomizationId());
+      urlBuilder.addQueryParameter(
+          "language_customization_id", recognizeOptions.languageCustomizationId());
     }
     if (recognizeOptions.acousticCustomizationId() != null) {
-      urlBuilder.addQueryParameter("acoustic_customization_id", recognizeOptions.acousticCustomizationId());
+      urlBuilder.addQueryParameter(
+          "acoustic_customization_id", recognizeOptions.acousticCustomizationId());
     }
     if (recognizeOptions.baseModelVersion() != null) {
       urlBuilder.addQueryParameter("base_model_version", recognizeOptions.baseModelVersion());
```

Comment:
I think you need to add `speech_detector_sensitivity` and `background_audio_suppression` to `urlbuilder`

Reply:
@mamoonraja are those parameters part of the query to the service during a websockey or part of the start message in the websocket?